djangocms_navigation/test_utils/factories.py

```diff
@@ -0,0 +1,142 @@
+import string
+
+from django.contrib.auth.models import User
+from django.contrib.contenttypes.models import ContentType
+from django.contrib.sites.models import Site
+
+from cms.models import Page, PageContent, TreeNode
+
+from djangocms_versioning.models import Version
+
+import factory
```

Comment:
isort might fail here.

Reply:
I think it does it alphabetically, no? Besides, I think this is copy and pasted from other factories.py files, so wouldn't that mean it fails everywhere?

Reply:
I thought all import should be a group on top and alphabetically sort

Reply:
Just tried running isort and it did something that looked ridiculous and made no sense to me :D (it also complained about 4 other files in the repo). I can commit the isort changed version if you prefer or I can just do what pep8 recommendation suggest (i.e. three import groups: https://www.python.org/dev/peps/pep-0008/#imports) or do nothing :P Please choose :D

Reply:
pep8 recommendation, please if you don't mind.

Reply:
not as expected but I will leave it for now and approve.